requirements/dev.txt

```diff
@@ -5,7 +5,6 @@
 #    make py-requirements
 #
 apipkg==1.5               # via execnet
-appnope==0.1.0            # via ipython
```

Comment:
need to figure out what this is about. maybe a difference between running pip-compile on linux and mac